### DISCUSSION TITLE
Error forwarding port  ddev codespaces

### DISCUSSION THREAD

**Discussion by shafqatDrupal** [community/community#48197] [Codespaces]
### Select Topic Area

Bug

### Body

When I open my site in a browser with HTTPS(not with HTTP) It responds with this error:
**Error forwarding port**

It was working find last week. Any idea what is possible issue?

**Comment by pamelafox** [comment]
I am also seeing a huge uptick in "Error forwarding port" in Codespaces that formerly worked all the time. My workaround right now is reloading and hoping for the best, but that really slows down the development experience. Is a regression causing an increase in forwarding errors, perhaps?

**Reply by jung-youjin** [reply]
Hi @pamelafox, facing the same issue as well without a workaround for now... 😢 Are you still having the same problem?
